File: one-statement/IzzyIguana.java
public class IzzyIguana {

    public static void main(String[] args) {
            
        System.out.println(new java.util.Scanner(System.in)
            .tokens()
            .mapToInt(java.util.Map.of("Lettuce", 5, "Carrot", 4, "Mango", 9, "Cheeseburger", 0)::get)
            .sum() >= 10 ? "Come on Down!" : "Time to wait");
    }
}

File: easy/pro-izzy-the-iguana/IzzyIguana.rb
snack_points = {Lettuce: 5, Carrot: 4, Mango: 9, Cheeseburger: 0}

totalSum = gets.split.map {|item| snack_points[item.to_sym]}.inject(:+)

print totalSum >= 10 ? 'Come on Down!' : 'Time to wait'
